include/dmlc/logging.h

```diff
@@ -14,6 +14,10 @@
 #include <stdexcept>
 #include "./base.h"
 
+#if DMLC_LOG_STACK_TRACE
+#include <cxxabi.h>
```

Comment:
I don't think this is available on all platforms. Have  you tried if it compiles on windows?

Reply:
I don't think this code works on windows as it is, that's why is protected with DMLC_LOG_STACK_TRACE which is not enabled as seen in base.h:

#if (defined(__GNUC__) && !defined(__MINGW32__)\
     && !(defined __MINGW64__) && !(defined __ANDROID__))
#if (!defined(DMLC_LOG_STACK_TRACE))
#define DMLC_LOG_STACK_TRACE 1
#endif

Reply:
In VS there is __unDName  but this would require additional implementation effort.